tests/e2e/configurer/chain/queries.go

```diff
@@ -255,3 +256,15 @@ func (n *NodeConfig) QueryListSnapshots() ([]*tmabcitypes.Snapshot, error) {
 
 	return listSnapshots.Snapshots, nil
 }
+
+// QueryTotalPools returns the total number of pools existing.
+func (n *NodeConfig) QueryTotalPools() uint64 {
```

Comment:
Same with this. It shows as merged already:
https://github.com/osmosis-labs/osmosis/pull/3504/files#diff-47c0a4d7ef6d0ef3de058bbba44963a3a94dfea4c2f0cd6be0bf05eb945f66dd